.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
```suggestion
      interval: "weekly"
```

Reply:
Okay, though "daily" would give quicker feedback if it worked, and we don't depend on so many actions that it's likely to be more than once a week anyway - but I have found you can manually trigger dependabot, so weekly is fine.